### DISCUSSION TITLE
About the blood and plasma PK

### DISCUSSION THREAD

**Discussion by Carrie-nothing** [Open-Systems-Pharmacology/Forum#1228] [Questions & Problems]
Hello tems,
When I adjusted the parameter value of "Blood/Plasma concentration ratio", the simulated PK curve did not change. May I ask why this was caused?

![1655197232(1)](https://user-images.githubusercontent.com/92715703/173538199-a63bbdf3-706a-454f-aa57-9ee09ac58f54.png)
![1655197313(1)](https://user-images.githubusercontent.com/92715703/173538464-29d71ec9-d5cb-4521-aef3-b2646f7098ae.png)

**Comment by StephanSchaller** [comment]
Dear @Carrie-nothing, It depends ;-)

See here:
https://github.com/Open-Systems-Pharmacology/Forum/discussions/897
